### PR TITLE
fix: Change new identity keys to hex format

### LINKED ISSUE #2770: `defradb identity new` returns keys in encoding not supported by defra calls
The private key returned from `defradb identity new` is binary encoded, whereas the defra only accepts hex-encoded identity private keys.

The encoding of the value returned from `defradb identity new` is also not documented.

### PR DESCRIPTION
**Pull request by shahzadlone**
## Relevant issue(s)
Resolves #2770 

## Description
- Return the priv/pub keys in hex

Specify the platform(s) on which this was tested:
- WSL2

### CODE REVIEW

**Review by AndrewSisley — APPROVED** [review]
LGTM, thanks Shahzad (I'm assuming you update the CLI docs before merge :))

**Comment by shahzadlone** [comment]
> LGTM, thanks Shahzad (I'm assuming you update the CLI docs before merge :))

Yup it's done.

**Pull request merged**
